src/app/item-popup/SocketDetails.m.scss

```diff
@@ -15,6 +15,7 @@
     display: flex;
     flex-direction: row;
     align-items: center;
+    min-height: 24px;
```

Comment:
We might also want to add `freezeInitialHeight` to the Sheet so it doesn't resize as you search.

Reply:
Added, but I also decided to cap the height of the sheet to a reasonable size because it looks a bit weird on my desktop screen when the shaders picker has a height of 1200px and wastes 1000px of these on blank space after filtering.